explorer_frontend/src/features/code/code-toolbar/OpenProjectButton.tsx

```diff
@@ -42,7 +42,7 @@ export const OpenProjectButton: FC<OpenProjectButtonProps> = ({ disabled }) => {
       {
         label: "Async call",
         onChange: () => {
-          updateRecentProjects();
+          codeLength > 0 && updateRecentProjects();
```

Comment:
It's better to be on guard level. Like sample with guard on codeLength that will affect in case then check this on component level

Reply:
ok